test/transmuxer.test.js

```diff
@@ -2484,3 +2488,59 @@ QUnit.test('buffers video samples until flushed', function() {
   QUnit.equal(segments[0].tags.audioTags.length, 0, 'generated no audio tags');
   QUnit.equal(segments[0].tags.videoTags.length, 2, 'generated two video tags');
 });
+
+QUnit.test('AAcStream parses correct ID3 tag size', function() {
```

Comment:
All of these tests should be in a `QUnit.module` named "AAC Stream". Then you can remove "AAcStream" from the test names.